modules/virt-about-instance-types.adoc

```diff
@@ -82,3 +82,10 @@ In addition to the required `cpu` and `memory` attributes, you can include the f
 `launchSecurity`:: Configure Secure Encrypted Virtualization (SEV).
 `nodeSelector`:: Specify node selectors to control the nodes where this VM is scheduled.
 `schedulerName`:: Define a custom scheduler to use for this VM instead of the default scheduler.
+
+[id="virt-about-instance-types-controller-revisions_{context}"]
+== Controller revisions
+
+When you create a VM by using an instance type, a `ControllerRevision` custom resource (CR) retains an immutable, snapshot copy of the instance type spec. This snapshot locks in resource-related characteristics defined in the instance type spec, such as the required guest CPU and memory. A reference to the `ControllerRevision` CR is contained in the VM spec.
```

Comment:
A `ControllerRevision` isn't a custom resource, if you're seeing it referred to as a `CR` in the upstream docs it's as an acronym for `ControllerRevision`. We also capture the complete object now, just just the spec.

Reply:
Should it just be "object" instead of custom resource? Or how should we refer to it?

Reply:
Thank you!